src/Microsoft.ML.Data/Transforms/HashTransform.cs

```diff
@@ -33,26 +33,48 @@ public sealed class HashTransform : OneToOneTransformBase, ITransformTemplate
         public const int NumBitsMin = 1;
         public const int NumBitsLim = 32;
 
+        private static class Defaults
+        {
+            public const int HashBits = NumBitsLim - 1;
+            public const uint Seed = 314489979;
+            public const bool Ordered = false;
+            public const int InvertHash = 0;
+        }
+
         public sealed class Arguments
         {
+            public Arguments()
+            {
+
+            }
+
+            public Arguments(string name, string source)
```

Comment:
I could see that in some circumstances, maybe having a specialized constructor on arguments could be useful, but why is it helpful here?